### DISCUSSION TITLE
2.1.1 broke updates - help

### DISCUSSION THREAD

**Discussion by yuriw** [Lissy93/dashy#1511] [Help and Support]
I used https://tteck.github.io/Proxmox/ script for Dashy/proxmox installation
After the 2.1.1 dir change the update does not work anymore https://github.com/tteck/Proxmox/discussions/2631

The new/updated script now uses a new dir structure.

I suspect if I install a fresh LCX the problem will be solved.

Now, how do I restore all settings from the current Dashy installation?
(I tried via the cloud backup/restore, but some widgets were not correctly restored)

TIA

**Comment by CrazyWolf13** [comment]
Has been resolved.

If the issue happens again, don't hesitate to create an issue, discussions often get overlooked.

**Comment by yuriw** [comment]
This may be unrelated but still unresolved => https://github.com/tteck/Proxmox/discussions/2631#discussioncomment-9461454
